entity-framework/ef6/querying/related-data.md

```diff
@@ -54,18 +54,18 @@ using (var context = new BloggingContext())
                        .Include(b => b.Posts.Select(p => p.Comments))
```

Comment:
This one is still not aligned.

Reply:
Wait, it isn't? Is it now?
Or do you want the "Select" below the "Posts"?